Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -534,3 +583,13 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         return df
     }()
 }
+
+private extension UICollectionView {
+    var isLastCellFullyVisible: Bool {
+        if numberOfItems(inSection: 0) == 0 { return true }
```

Comment:
Do we need this check? 🤔 The layout returns `nil` when asking for the attributes, I guess (didn't test it)

Reply:
The guard on line 592 return `false`, where this check returns `true`.
A better question is: How it should behave when there are no messages (not loaded yet) in the message list? 
Considering that the message list will open scrolled to the latest message (the last cell will be visible) I'm returning `true`.